backend/models/index.js

```diff
@@ -8,6 +8,8 @@ const env = process.env.NODE_ENV || "development";
 const config = require(__dirname + "/../config/database")[env];
 const db = {};
 
+const postgresConfig = require(__dirname + "/../config/database")["postgres"];
+
```

Comment:
```suggestion
const dbConfig = require(__dirname + "/../config/database");
const db = {};

```

Then, use `dbConfig.sqlite` and `dbConfig.postgres` where applicable.